apps/web/getSubdomainRegExp.js

```diff
@@ -0,0 +1,15 @@
+const getDefaultSubdomain = (url) => {
+  if (!url.startsWith("http:") && !url.startsWith("https:")) {
+    // Make it a valid URL. Mabe we can simply return null and opt-out from orgs support till the use a URL scheme.
```

Comment:
Someone reported they were using cal.com without specifying https:// earlier. So I just added it, didn't verify what else fails if someone uses that.